link/uprobe_multi.go

```diff
@@ -0,0 +1,148 @@
+package link
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"unsafe"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/asm"
+	"github.com/cilium/ebpf/internal"
+	"github.com/cilium/ebpf/internal/sys"
+	"github.com/cilium/ebpf/internal/unix"
+)
+
+type UprobeMultiOptions struct {
+	Path          string
+	Offsets       []uint64
+	RefCtrOffsets []uint64
+	Cookies       []uint64
+}
+
+func (ex *Executable) UprobeMulti(symbols []string, prog *ebpf.Program, opts *UprobeMultiOptions) (Link, error) {
+	if opts == nil {
+		opts = &UprobeMultiOptions{}
+	}
+	var err error
+	if len(symbols) != 0 {
```

Comment:
Is it valid to not specify any symbols? And why do we need to avoid calling addressMulti if there aren't any?

Reply:
yes, it's valid because all the necessary info to attach can be provided by UprobeMultiOptions